src/autora/doc/pipelines/main.py

```diff
@@ -108,6 +108,18 @@ def import_model(model_name: str) -> None:
     pass
 
 
+@app.command()
+def import_data(code_file: str, text_file: str, output_file: str = "data.jsonl") -> None:
+    from pathlib import Path
+
+    import jsonlines
+
+    # alpaca jsonl format:
+    d = {"instruction": Path(code_file).read_text(), "output": Path(text_file).read_text()}
```

Comment:
```suggestion
    def read_text(file: str) -> str:
        return Path(file).read_text()
    d = {"instruction": read_text(code_file), "output": read_text(text_file)}
```

Just a small suggestion for a little bit of readability for the dictionary, plus now it's set up that if you maybe have these inputs in other place than local or other types of validations, you can just modify the function `read_text` 😄